### ISSUE TITLE
Careerfy Theme/ WP JobSearch plugin conflict 

### ISSUE THREAD

**Issue by ernee** [google/site-kit-wp#1538]
## Bug Description

The user [in this support topic](https://wordpress.org/support/topic/uncaught-google_auth_exception/) reported a fatal error when trying to setup Site Kit while using the [Careerfy Theme](https://themeforest.net/item/careerfy-job-board-wordpress-theme/21137053) and its bundled plugin WP JobSearch. 

```
Error Details
=============
An error of type E_ERROR was caused in line 125 of the file /home/401569.cloudwaysapps.com/jgxzyhraqb/public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google-sdk/Auth/OAuth2.php. Error message: Uncaught Google_Auth_Exception: Error fetching OAuth2 access token, message: ‘invalid_request: Could not determine client ID from request.’ in /home/401569.cloudwaysapps.com/jgxzyhraqb/public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google-sdk/Auth/OAuth2.php:125
Stack trace:
#0 /home/401569.cloudwaysapps.com/jgxzyhraqb/public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google-sdk/Client.php(128): Google_Auth_OAuth2->authenticate(‘sitekit-afePeke…’, false)
#1 /home/401569.cloudwaysapps.com/jgxzyhraqb/public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google.php(89): Google_Client->authenticate(‘sitekit-afePeke…’)
#2 /home/401569.cloudwaysapps.com/jgxzyhraqb/public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google.php(50): JobsearchGoogle->set_access_tokes()
#3 /home/401569.cloudwaysapps.com/jgxzyhraqb/public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google.php(544): JobsearchGoogle->__construct()
```

In testing, I encountered the following screen at the end of the setup process when being redirected to the dashboard (from the "congratulations" page): 

<img width="500" alt="Careerfy: WP JobSearch" src="https://user-images.githubusercontent.com/8108643/81465076-258a1500-917c-11ea-9170-35f87fba7a8b.png">

At the following callback URL:

https://example.com/wp-admin/index.php?oauth2callback=1&code=sitekit-SJvvcDCZBVEdZQbuka5YHa

And received a fatal error in the WP debug log:

```
PHP Fatal error:  Uncaught Google_Auth_Exception: Error fetching OAuth2 access token, message: 'invalid_grant: Malformed auth code.' in /home/.../public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google-sdk/Auth/OAuth2.php:125
Stack trace:
#0 /home/.../public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google-sdk/Client.php(128): Google_Auth_OAuth2->authenticate('sitekit-SJvvcDC...', false)
#1 /home/.../public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google.php(89): Google_Client->authenticate('sitekit-SJvvcDC...')
#2 /home/.../public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google.php(50): JobsearchGoogle->set_access_tokes()
#3 /home/.../public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google.php(544): JobsearchGoogle->__construct()
#4 /home/.../public_html/wp-content/plugins/wp-jobsearch/modules/social-login/social-login.php(26): include('/home/.../...')
#5 /home/.../public in /home/.../public_html/wp-content/plugins/wp-jobsearch/modules/social-login/google/google-sdk/Auth/OAuth2.php on line 125
```
## Steps to reproduce

1. Install and activate the [Careerfy Theme ](https://themeforest.net/item/careerfy-job-board-wordpress-theme/21137053)and bundled plugin WP JobSearch.
2. Activate Site Kit and proceed with the setup process
3. Reach the error page instead of the dashboard from the "congratulations" page 

## Screenshots

WP JobSearch has options for the Google API in its settings:

<img width="889" alt="WP JobSearch Google API section" src="https://user-images.githubusercontent.com/8108643/81465358-77339f00-917e-11ea-9738-bc40a526259e.png">

## Additional Context

 - PHP Version: 7.2.30
 - OS: MacOs
 - Browser: Chrome
 - Plugin Version: 1.8.0
 - Device: MacBook Air

I attempted the setup process with the WP JobSearch plugin deactivated and was able to proceed. However, after reactivation and attempting to connect Analytics, the error was encountered again on the callback from the OAuth.

related support topic: https://wordpress.org/support/topic/uncaught-google_auth_exception/ (Site Health submitted through form) 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## QA Brief

* 

## Changelog entry

*

**Comment by ernee** [comment]
Update: theme/ plugin authors have found a resolution for the issue and it will be in their next release.

**Comment by ernee** [comment]
Issue fixed in version 3.9.0 of theme.
